CSV: Set operator-type: non-standalone

dprince opened the pull request:
This should hide the placement package/bundle in the OpenShift console as we only want the main OpenStack operator to show up there.

stuggi commented on api/v1beta1/placementapi_types.go:
```diff
@@ -182,3 +182,13 @@ func (instance PlacementAPI) GetEndpoint(endpointType common.Endpoint) (string,
+	// the service is registered in keystone
+	// AND
+	// there is at least a single pod service the placement service
+	return instance.Status.ServiceID != "" && instance.Status.ReadyCount >= 1
```
this might change when we have conditions added

abays reviewed (approved):
/lgtm

dprince committed:
CSV: Set operator-type: non-standalone

This should hide the placement package/bundle in the OpenShift console
as we only want the main OpenStack operator to show up there.

gibizer commented:
/test placement-operator-build-deploy
Test executor was unable to claim a test cluster
